test/backend_test.in.cpp

```diff
@@ -9371,6 +9371,66 @@ NGRAPH_TEST(${BACKEND_NAME}, argmax_trivial)
     EXPECT_EQ((vector<int>{1, 3, 0}), read_vector<int>(result));
 }
 
+NGRAPH_TEST(${BACKEND_NAME}, argmax_3D_axis_0) // Along Channels
+{
+    Shape shape{3, 4, 2}; // CHW ->(0,1,2)
+    Shape rshape{4, 2};
+    auto A = make_shared<op::Parameter>(element::f32, shape);
+    auto f =
+        make_shared<Function>(make_shared<op::ArgMax>(A, 0, element::i32), op::ParameterVector{A});
+
+    auto backend = runtime::Backend::create("${BACKEND_NAME}");
+
+    // Create some tensors for input/output
+    auto a = backend->create_tensor(element::f32, shape);
+    copy_data(a, vector<float>{8, 4, 12, 10, 2, 9, 1, 5, 6, 7, 11, 3,
```

Comment:
if you want, you can lay out 342 like this
```
xx
xx
xx
xx

xx
xx
xx
xx

xx
xx
xx
xx
```
here is an example:
https://github.com/NervanaSystems/ngraph/blob/2b289df09692b0989873229ed745787ea8881647/test/backend_test.in.cpp#L5159

easier to read.

Reply:
same of other tests.